src/ai/backend/storage/filebrowser/database.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any
+
+import sqlalchemy as sa
+
+
+class FilebrowserTrackerDB:
+    def __init__(self, db_path: str):
+        self.meta = sa.MetaData()
+        self.containers = sa.Table(
+            "containers",
+            self.meta,
+            sa.Column("container_id", sa.String, primary_key=True),
+            sa.Column("container_name", sa.String),
+            sa.Column("service_ip", sa.String),
+            sa.Column("service_port", sa.Integer),
+            sa.Column("config", sa.Text),
+            sa.Column("status", sa.String),
+            sa.Column("timestamp", sa.String),
+        )
+        self.db_path = db_path
+        self.url = f"sqlite:///{str(db_path)}"
+        self._engine = sa.engine
+        self.engine = sa.create_engine(self.url)
+
+        insp = sa.inspect(self.engine)
+        if "containers" not in insp.get_table_names():
+            self.meta.create_all(self.engine)
+
+    async def get_all_containers(self) -> Any:
+        with self.engine.connect() as connection:
+            rows = connection.execute(self.containers.select())
```

Comment:
These calls use blocking I/O, so should be executed with `loop.run_in_executor()`. Or, even better, we can utilize async version of sqlite, such as `aiosqlite`... Please make up your mind and either replace all sqlite calls to use `aiosqlite` or wrap it under `run_in_executor()`.